src/mongocxx/options/find.hpp

```diff
@@ -26,13 +26,14 @@ namespace mongocxx {
 MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
+enum class cursor_type : std::uint8_t { k_non_tailable, k_tailable, k_tailable_await };
```

Comment:
Too many things were named `cursor_type`. At the time, I couldn't get this to work without pulling it out of the same scope as all of the other things named this. I'll try again and let you know the exact issue.